etc/initdev/Vagrantfile

```diff
@@ -4,7 +4,7 @@
 VAGRANTFILE_API_VERSION = "2"
 
 Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
-  config.vm.box = "ubuntu/vivid64" # 15.04
+  config.vm.box = "bento/ubuntu-16.04"
```

Comment:
Cool. What was it that lead you to the vagrant environment? We should probably correct that to nudge people toward the docker flow. Or was it just the existence of the vagrant file?